vnext/Microsoft.ReactNative/Views/UnimplementedViewManager.cpp

```diff
@@ -59,9 +59,11 @@ const wchar_t *UnimplementedViewManager::GetName() const {
 void UnimplementedViewManager::GetNativeProps(const winrt::Microsoft::ReactNative::IJSValueWriter &writer) const {
   Super::GetNativeProps(writer);
 
+  // Modal View Props (Todo: fix so these aren't in UnimplementedView)
```

Comment:
What's the timing for this TODO? An issue tracking you can link here?
(As a rule, don't check in TODO comments without tracking)

Reply:
I can make an issue :) This is actually a bigger issue than just Modal, I'm not sure if we want to force Fabric components to also have Views in Paper but it will take some work to figure that out